reference-architecture/aws-ansible/playbooks/roles/etcd-volume/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Create filesystem for /var/lib/etcd
+  filesystem:
+      fstype: xfs
+      dev: /dev/xvdc
+
+- name: Create local volumes directory
+  file:
+      path: /var/lib/etcd
+      state: directory
+      recurse: yes
+
+- name: Create fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: present
+
+- name: Mount fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: mounted
```

Comment:
I believe the long term plan is for the block storage device to become a variable so that in the future when we move to common roles for all providers then providers like vmware that can't support user-data will be covered as well.